client/packages/common/src/hooks/useNativeClient/useNativeClient.ts

```diff
@@ -1,6 +1,6 @@
 import { useState, useEffect, useCallback } from 'react';
 import { useMutation } from 'react-query';
-import { uniqWith } from 'lodash';
```

Comment:
it's ok to import when in the common package, but for consistency, and to prevent copy-paste errors later, I've updated em all.